server/controller/routes/getQuestions.js

```diff
@@ -0,0 +1,38 @@
+const getQuestions = require('../../db/queries/getQuestions')
+const  splitString = require('../utils/splitString')
+
+module.exports = async (req, res, next) => {
+
+  let newQuestionsArray = []
```

Comment:
delete this line there is no need for it 
you can put const before line 12 directly

Reply:
Done